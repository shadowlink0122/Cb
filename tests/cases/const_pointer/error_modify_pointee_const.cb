// エラーテスト: const T* 経由で値を変更しようとする
include "stdio.cb"

int main() {
    println("=== Test: Cannot modify through pointer to const ===");
    int x = 10;
    const int* ptr = &x;
    
    // エラー: const int* 経由では値を変更できない
    *ptr = 20;  // This should fail
    
    println("ERROR: This should not execute");
    return 0;
}
