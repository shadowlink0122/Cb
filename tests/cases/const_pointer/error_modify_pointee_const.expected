=== Test: Cannot modify through pointer to const ===
Error: Cannot modify value through pointer to const (const T*)
